src/components/Login/Login.js

```diff
@@ -0,0 +1,51 @@
+//Imports
+import React, { useState } from 'react';
+
+//UI Imports
+import './Login.scss';
+
+const Login = () => {
+  const [validLogin, setValidLogin] = useState(true)
+  const [username, setUsername] = useState('')
+  const [password, setPassword] = useState('')
+
+  //default is true! So no error message appears
+  //if value is blank, then setValidLogin(false)
+  //display error message
+  //include minvalue for security
+
+  const checkLogin = (event) => {
+    if ( username.length > 6 && password.length > 6 ) {
+      event.preventDefault();
+      setValidLogin(false);
+      console.log(validLogin)
+
```

Comment:
For MVP, will we be sending a request to the BE on login check? What will the query look like (send user name and get user data as response?), or verify username existence and query user data on user landing page load?